src/platform/silabs/efr32/rs911x/BLEManagerImpl.cpp

```diff
@@ -252,11 +252,11 @@ CHIP_ERROR BLEManagerImpl::_Init()
     mServiceMode = ConnectivityManager::kCHIPoBLEServiceMode_Enabled;
 
     // Create FreeRTOS sw timer for BLE timeouts and interval change.
-    sbleAdvTimeoutTimer = xTimerCreate("BleAdvTimer",             // Just a text name, not used by the RTOS kernel
-                                       BLE__DEFAULT_TIMER_PERIOD, // == default timer period (mS)
-                                       false,                     // no timer reload (==one-shot)
-                                       (void *) this,             // init timer id = ble obj context
-                                       BleAdvTimeoutHandler       // timer callback handler
+    sbleAdvTimeoutTimer = xTimerCreate("BleAdvTimer",                            // Just a text name, not used by the RTOS kernel
+                                       pdMS_TO_TICKS(BLE__DEFAULT_TIMER_PERIOD), // == default timer period (mS)
```

Comment:
The comment is incorrect, this parameter is specified in ticks, not mS.

Reply:
done